src/components/ProductDetails/ProductDetails.component.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import './ProductDetails.component.scss';
+import endpoints from '../../api/endpoints.config';
+import { Button, Modal, ModalHeader, ModalBody, ModalFooter } from 'react-bootstrap';
+import RatingStarsComponent from '../RatingStars/RatingStars.component';
+
+class ProductDetailsComponent extends React.Component {
```

Comment:
Converted ProductDetails as a functional component.

API Call to get the product specific details using product ID is not available, so displaying them in Modal instead of new screen.